.github/workflows/test.yaml

```diff
@@ -30,10 +62,10 @@ jobs:
           sudo apt update
           sudo apt build-dep -d -y ${{ github.workspace }}/src/github.com/snapcore/snapd
     # Work around caching files owned by root https://github.com/actions/cache/issues/133
-    - name: Make /var/cache/apt owned by current user
-      run: sudo chown -R $(id -u) /var/cache/apt
     - name: Install Debian dependencies
       run: sudo apt build-dep -y ${{ github.workspace }}/src/github.com/snapcore/snapd
+    - name: Make /var/cache/apt owned by current user
+      run: sudo chown -R $(id -u) /var/cache/apt
     - name: Set up go 1.9
       uses: actions/setup-go@v1
       with:
```

Comment:
I think the unit test step should either run "./run-checks --unti" and we fix whatever needs fixing in this script. If not I think we should remove the "--unit" step from run-checks to avoid things drifting appart.

Reply:
I did this in the follow-up to this (closed) PR.